tests/functional/src/main/java/org/mule/functional/classloading/isolation/maven/MavenMultiModuleArtifactMapping.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.functional.classloading.isolation.maven;
+
+/**
+ * Defines the multi-module folder name for an artifactId. It is useful when the convention of using the same artifactId
```

Comment:
This javadoc is not clear to me. I would describe why this is needed and what conditions must be meet on the multi module project to make it work.